src/components/time-input/time-input.e2e.ts

```diff
@@ -104,7 +104,8 @@ describe('sbb-time-input', () => {
     expect(input).not.to.have.attribute('data-sbb-invalid');
   });
 
-  it('should interpret valid values', async () => {
+  it('should interpret valid values', async function () {
+    this.timeout(5000);
```

Comment:
what's the difference between this.timeout and aTimeout?

Reply:
`aTimeout` waits for time
`this.timeout(5000)` sets the maximum time that the test can last before being forced to failure ( overrides the global config we have in `web-test-runner.config` )

Reply:
thanks for the hint